autoconnect/autoconnect-ws/autoconnect-ws-sm/src/unidentified.rs

```diff
@@ -115,8 +115,8 @@ impl UnidentifiedClient {
             self.ua,
             broadcast_subs,
             flags,
-            user.connected_at,
-            user.current_timestamp,
+            user.connected_at_ms,
+            user.current_timestamp_ms,
```

Comment:
`current_timestamp` is indeed `ms` here as it's read from the database, having been written there from `increment_storage`, which wrote a `sortkey_timestamp` (`ms`) there.